api/src/main/java/org/openmrs/module/ModuleUtil.java

```diff
@@ -67,7 +67,7 @@ private ModuleUtil() {
 	 *
 	 * @param props Properties (OpenMRS runtime properties)
 	 */
-	public static void startup(Properties props) throws ModuleMustStartException, OpenmrsCoreModuleException {
+	public static void startup(Properties props) throws ModuleMustStartException {
```

Comment:
Shouldn't instead be leaving the more specific exception like OpenmrsCoreModuleException?